### PR TITLE
[Fix] Make `resolveProposalJustification` safer with white-listing domains

### LINKED ISSUE #61: In `resolveProposalJustification` setup a whitelist of allow domains for justification
This is to ensure councils are not being exposed to malicious links

List
- `medium.com`
- `hackmd.io`

### PR DESCRIPTION
**Pull request by ken-futureverse**
## Summary

Make `resolveProposalJustification` safer with white-listing domains plus fix build errors due to double slashes `ipfs//`

Closes #61

### CODE REVIEW

File: libs/node/utils/src/resolveProposalJustification.ts [hunk 1]
```diff
@@ -10,10 +10,55 @@ import { safeFetch } from "./";
...
+	if (uriType === "external") return fallback;
+
+	const pinataUri = fixPinataGateway(justificationUri);
+
+	let response;
+	try {
+		response = await safeFetch(pinataUri);
+	} catch (error) {
+		console.error(error);
+	}
```
Comment (aidan-starke) [inline libs/node/utils/src/resolveProposalJustification.ts:28]:
```suggestion
	let response;
	try {
	  const pinataUri = fixPinataGateway(justificationUri);
		response = await safeFetch(pinataUri);
	} catch (error) {
		console.error(error);
	}
```

Comment (aidan-starke) [inline libs/node/utils/src/resolveProposalJustification.ts:28]: Proposal 44 is failing as its URL is "0x00"

**Review by aidan-starke — CHANGES_REQUESTED** [review]

File: libs/node/utils/src/resolveProposalJustification.ts (hunk 1 above)
Comment (ken-futureverse) [reply libs/node/utils/src/resolveProposalJustification.ts:28]: yeah I might put some stuff around that to ignore it, and resolve the conflict 👇 of course

Comment (ken-futureverse) [reply libs/node/utils/src/resolveProposalJustification.ts:28]: sorted!

**Review by aidan-starke — APPROVED** [review]

**Pull request merged**